src/components/GlobalPreferences/Notifications/NotificationsLogin.js

```diff
@@ -1,72 +1,100 @@
 import React, { useState } from 'react'
 import styled from 'styled-components'
 import PropTypes from 'prop-types'
-import { Box, Button, GU, TextInput, Info, IconMail } from '@aragon/ui'
+import {
+  Box,
+  Button,
+  GU,
+  TextInput,
+  Info,
+  IconMail,
+  Text,
+  useTheme,
+} from '@aragon/ui'
 import { login } from './notification-service-api'
-import { getEthNetworkType } from '../../../local-settings'
+import { network } from '../../../environment'
+import { validateEmail } from '../../../utils'
 import notificationSvg from './notifications.svg'
 
-export default function NotificationsLogin({
-  dao,
-  authState,
-  email,
-  onEmailChange,
-}) {
+export default function NotificationsLogin({ dao, authState, onEmailChange }) {
   const [inputEmail, setInputEmail] = useState('')
+  const [emailInvalid, setEmailInvalid] = useState(false)
   const [apiError, setApiError] = useState(null)
-  const network = getEthNetworkType()
+  const ethNetwork = network.type === 'main' ? 'mainnet' : 'rinkeby'
```

Comment:
We should leave a comment here about the API server requiring `mainnet`.

I was also thinking if we should use [`sanitizeNetworkType()`](https://github.com/aragon/aragon/blob/master/src/network-config.js#L74) for this, but think it's better not to in case someone changes that function later to make strings "prettier".